### PR TITLE
ssh*: Add German translation

### PR DESCRIPTION
**Pull request by flyck**
- [X] The page (if new), does not already exist in the repo.
- [X]  The page is in the correct platform directory (common/, linux/, etc.)
- [X]  The page has 8 or fewer examples.
- [X]  The PR title conforms to the recommended templates.
- [X]  The page follows the content guidelines.
- [X]  The page description includes a link to documentation or a homepage (if applicable).

Hello again!

I translated all the ssh pages into German.

### CODE REVIEW

File: pages.de/common/ssh-keygen.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+`ssh-keygen -t ed25519 -a 100`
+
+- Generieren eines 4096 Bit langen RSA Schlüssel-Paars mit der Email im Kommentarfeld:
+
+`ssh-keygen -t rsa -b 4096 -C "{{Email}}"`
+
+- Abrufen des Schlüssel Fingerabdrucks von einem Server (hilfreich um die Authentizität eines Servers beim ersten Verbinden zu überprüfen):
+
+`ssh-keygen -l -F {{Externer-Server}}`
```
Comment (navarroaxel) [inline pages.de/common/ssh-keygen.md:23]: We use `snake_case` for variable names. Can we use this format in all pages?
```suggestion
`ssh-keygen -l -F {{Externer_Server}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @flyck, I left 1 comment.

File: pages.de/common/ssh-keygen.md (hunk 1 above)
Comment (flyck) [reply pages.de/common/ssh-keygen.md]: Of course 👍

**Review by navarroaxel — APPROVED** [review]
Thanks @flyck, please **don't** push force, it's harder to us to track the changes and suggestions.

**Comment by flyck** [comment]
Ah right, my bad, I'll keep it in mind for future PRs!

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Comment by sbrl** [comment]
Also next time, perhaps you could avoid force-pushing please? It makes it harder to tell what's changed when reviewing, and we've got a special button that allows us to squash easily on merge :smiley_cat:

**Pull request merged**